### ISSUE TITLE
Exception when input does not have percentile at exactly 0.5

### ISSUE THREAD

**Issue by 0xE1E10** [BrunoBonacci/hdr-plot#2]
My input data does not have a percentile at exactly 0.5. It was at 0.52 for some reasons.

```
Traceback (most recent call last):
  File ".venv/bin/hdr-plot", line 18, in <module>
    plot.main()
  File "/opt/intel/flexran/dpdk/vmwaretools/.venv/lib/python3.7/site-packages/hdr_plot/hdr_plot.py", line 110, in main
    plot_summarybox(ax, pct_data, labels)
  File "/opt/intel/flexran/dpdk/vmwaretools/.venv/lib/python3.7/site-packages/hdr_plot/hdr_plot.py", line 60, in plot_summarybox
       Value     Percentile TotalCount 1/(1-Percentile)

        0.00 0.000000000000         19           1.00
       59.00 0.100000000000      15609           1.11
       76.00 0.190000000000      29483           1.23
       83.00 0.271000000000      41776           1.37
       86.00 0.343900000000      56308           1.52
       87.00 0.409510000000      62278           1.69
       89.00 0.468559000000      74214           1.88
       90.00 0.521703100000      79637           2.09
       92.00 0.569532790000      89321           2.32
       93.00 0.612579511000      94175           2.58
... (rest of the file)

    textstr = '\n'.join([info_text(labels[i], percentiles[i]) for i in range(len(labels))])
  File "/opt/intel/flexran/dpdk/vmwaretools/.venv/lib/python3.7/site-packages/hdr_plot/hdr_plot.py", line 60, in <listcomp>
    textstr = '\n'.join([info_text(labels[i], percentiles[i]) for i in range(len(labels))])
  File "/opt/intel/flexran/dpdk/vmwaretools/.venv/lib/python3.7/site-packages/hdr_plot/hdr_plot.py", line 46, in info_text
    "max",    data['Latency'].max())
  File "/opt/intel/flexran/dpdk/vmwaretools/.venv/lib/python3.7/site-packages/pandas/core/series.py", line 139, in wrapper
    raise TypeError(f"cannot convert the series to {converter}")
TypeError: cannot convert the series to <class 'float'>
```

Offending line is the median calculation:
```
def info_text(name, data):
    textstr = '%-18s\n------------------\n%-6s = %6.2f ms\n%-6s = %6.2f ms\n%-6s = %6.2f ms\n'%(
        name,
        "min",    data['Latency'].min(),
        "median", data[data["Percentile"] == 0.5]["Latency"],
        "max",    data['Latency'].max())
    return textstr
```

I don't know why we're trying to calculate this manually but simply calling the median function fixes the problem.

```
def info_text(name, data):
    textstr = '%-18s\n------------------\n%-6s = %6.2f ms\n%-6s = %6.2f ms\n%-6s
 = %6.2f ms\n'%(
        name,
        "min",    data['Latency'].min(),
        "median", data['Latency'].median(),
        "max",    data['Latency'].max())
    return textstr
```

**Comment by BrunoBonacci** [comment]
Hi,

Thanks for reporting this, and thanks for the suggested fix, but unfortunately, your solution would produce the wrong result.

`data['Latency'].median()` would return the median of the values in the file, however a "hdr-file" contains the reported latency skewed towards the high percentiles. 
As you can clearly see from the following sample file:
https://github.com/BrunoBonacci/hdr-plot/blob/master/sample/file3.out#L27
the median is `1.001 ms`, while for the median returned by `data['Latency'].median()` is `2.295 ms`.

I've now published the version `0.2.3` which should fix the above issue, please do let me know if it works for you.

**Comment by BrunoBonacci** [comment]
releases with `v0.2.3`
install with `pip install hdr-plot --upgrade`

**Comment by 0xE1E10** [comment]
You're right. Now I understand why you can't just use median(). Because it's blissfully unaware of the counts/percentiles and act like all values have a count of 1. I think your fix is reasonable for when the 0.5 point is missing. I tried it on my data set just now. Worked flawlessly. Thanks.

Also, noticed that this new version has pandas listed in setup.py instead of asking people to install pandas separately. I was about to ask for that next. Now I can remove the hacks in my auto deps generation tool. Thanks.
